SIL.Linux.Logging/SIL.Linux.Logging.csproj

```diff
@@ -4,12 +4,12 @@
     <RootNamespace>SIL.Linux.Logging</RootNamespace>
     <AssemblyTitle>SIL.Linux.Logging</AssemblyTitle>
     <Description>SIL.Linux.Logging provides a library to log to the syslog service.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="Mono.Posix" Version="5.4.0.201" Condition="'$(OS)' == 'Windows_NT'" />
```

Comment:
You should also remove the `Mono.Posix` reference on Linux a few lines below.

I'm not sure where this assembly is used, but for compatibility it might be better to additionally compile SIL.Linux.Logging for `net461` so that we don't introduce an possibly incompatible different `Mono.Posix` dependency.

Reply:
Ok, thanks, looking into this now